app/helpers/icons_helper.rb

```diff
@@ -1,15 +1,20 @@
 module IconsHelper
   def graphical_icon(icon)
-    icon(icon, nil)
+    icon(icon, nil, role: :presentation)
```

Comment:
You could consider forcing the role if no alt if given. (no alt isn't the same as an empty alt).